src/main/java/org/tc/mail/MailNotificationSender.java

```diff
@@ -24,10 +26,15 @@
     private static final String NEW_COURSE_NOTIFICATION_SUBJECT = "New Course Added";
     private static final String REJECTED_COURSE_NOTIFICATION_SUBJECT = "Course rejected";
     private static final String DELETED_COURSE_NOTIFICATION_SUBJECT = "Course Deleted";
```

Comment:
It is good to use message source for email subjects so you can localize notification for different languages.